### ISSUE TITLE
Tons of plots appearing while fitting

### ISSUE THREAD

**Issue by jmrichardson** [mljar/mljar-supervised#451]
I am on windows using pycharm IDE following a tutorial an example tutorial.  When I fit the model, I get a bunch of figures that pop up on the screen and then it errors out with a pixelmap error.  Here is a screen [shot](url)

![image](https://user-images.githubusercontent.com/6565528/128730646-3291f060-f875-4542-99ee-a5886c238019.png)

Thanks for any help

**Comment by jmrichardson** [comment]
I set explain_level=0 and problem resolved

**Comment by pplonski** [comment]
@jmrichardson thanks for reporting. This looks like a bug - I would like to look closer on it. All figures should be plotted to file and displayed in `README.md` file.

**Comment by jmrichardson** [comment]
I just restarted my PC, which of course restarted PyCharm, to see if it was something with pycharm or my PC.  Fortunately, it fixed the problem of plots appearing and they do appear in the readme.  Sorry for any inconvenience but don't think it's an issue with mljar.  And by the way, I have only just started playing with mljar and it is REALLY nice!  Thanks so much for this package!  So much easier and intuitive than anything else I have used!
